### PR TITLE
SDT-77: Establish Connectivity b/w SDT & MCOL

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by timja — DISMISSED** [review]

File: apps/civil/civil-sdt/civil-sdt.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+metadata:
+  name: civil-sdt
+  namespace: civil
+spec:
+  releaseName: civil-sdt
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/civil/sdt:prod-f7b9f73-20230120155915 #{"$imagepolicy": "flux-system:civil-sdt"}
```
Comment (paulridout) [inline apps/civil/civil-sdt/civil-sdt.yaml:12]: The image tag (prod-f7b9f73-20230120155915) is one of the civil-sdt-gateway ones.  It should be the latest civil-sdt prod image tag from the [hmctspublic.io container registry](https://portal.azure.com/#blade/Microsoft_Azure_ContainerRegistries/RepositoryBlade/id/%2Fsubscriptions%2F8999dec3-0104-4a27-94ee-6588559729d1%2FresourceGroups%2Frpe-acr-prod-rg%2Fproviders%2FMicrosoft.ContainerRegistry%2Fregistries%2Fhmctspublic/repository/civil%2Fsdt)

Comment (kiran-yenigala-hmcts) [reply apps/civil/civil-sdt/civil-sdt.yaml:12]: Will resolve this issue once Demo branch is working fine

**Review by paulridout — APPROVED** [review]
Changes look OK (Comment about image will be addressed later).

**Review by paulridout — APPROVED** [review]
Changes look OK.

**Review by danlysiak — APPROVED** [review]

**Pull request merged**